src/manual/tasks/runs.md

```diff
@@ -0,0 +1,36 @@
+---
+filename: tasks/runs.md
+title: Runs
+order: 30
+---
+
+A task definition is, in a sense, the input to task execution. The output comes
+in the form of "runs" and associated "artifacts".
+
+A run is an attempt to execute a task. Most tasks have a single run, but in
+cases where transient errors such loss of a compute node cause a run to fail,
+Taskcluster automatically adds additional runs.
+
+Runs are identified by their `runId`, a small integer. While for most tasks,
+the single `runId` is `0`, you should always use the latest run to avoid
+reading information about a run that failed.
+
+Each run has properties describing the worker that claimed the run (referenced
+by the two-part namespace `<workerGroup>/<workerId>`), the reason the run was
+created, its current status, and various timestamps.
+
+A run is in one of a few states:
+
+ * `pending` - ready to be claimed and executed
+ * `running` - currently executing
+ * `completed` - finished successfully
+ * `failed` - finished unsuccessfully
+ * `exception` - finished due to some reason internal to Taskcluster
```

Comment:
I'll expand this a little, and leave a pointer to the queue reference for details.